src/App.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
-import './App.css';
+import { useSelector } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
+import SplashScreen from './components/SplashScreen/SplashScreen';
 import Register from './components/Register/Register';
 import Login from './components/Login/Login';
+import Doctors from './components/Doctors/Doctors';
+import BookAppointment from './components/Appointments/BookAppointment';
+import MyAppointments from './components/Appointments/MyAppointments';
+import Navbar from './components/layouts/Navbar';
+import AddDoctor from './components/Doctors/AddDoctor';
+import DeleteDoctor from './components/Doctors/DeleteDoctor';
+import DoctorDetails from './components/Doctors/DoctorDetails';
 
 function App() {
```

Comment:
- [ ] OPTIONAL: The project is still not responsive on small devices:

![image](https://user-images.githubusercontent.com/57421823/207301017-80f4e03e-59f8-444e-95c5-3dc490073f25.png)

I highly suggest you correct this issue so your app runs and shows as intended. 👍